test/libsolidity/syntaxTests/operators/custom/operator_bind_on_file_and_contract_level.sol

```diff
@@ -0,0 +1,28 @@
+type Int is int;
+
+using {add as +} for Int;
+
+function add(Int, Int) pure returns (Int) {
+    return Int.wrap(0);
+}
+
+function another_add(Int, Int) pure returns (Int) {
+    return Int.wrap(0);
+}
+
+contract B {
+    using {another_add as +} for Int;
+
+    function f() public {
+        Int.wrap(0) + Int.wrap(0);
+    }
+}
+
+contract C is B {
+    function g() public {
+        Int.wrap(0) + Int.wrap(0);
+    }
+}
+
+// ----
+// TypeError 2271: (281-306): Binary operator + not compatible with types Int and Int. Multiple user-defined functions provided for this operator.
```

Comment:
Like I mentioned earlier, we should have secondary locations pointing at the operator definitions for errors in the `using` statement. But in cases like this it would be useful to have them too. When compiling this example on the console, it should show the two `using` directives as the sources of those multiple definitions.